### DISCUSSION TITLE
STELLA... but for Indoor/Outdoor Air Quality???

### DISCUSSION THREAD

**Discussion by TerraEPO** [STELLA-Landsat/STELLA#8] [Ideas]
I’m currently pushing local school district leaders to quickly clean up the air in classrooms, and as a result have been developing an “indoor air quality (AQ) & how outdoor AQ impacts it” citizen science proposal that has students collect data both inside and outside of schools, while incorporating tools like Aranet4 (CO2 monitor), PM2.5, and other handheld AQ monitors.
 
I absolutely LOVE the STELLA’s design (and that students can build it), the bluetooth capabilities, and the built in visualization tool, and am wondering how we can leverage (*screams*) STELLAAA in these efforts. 

One novel approach being used with many low-cost air sensors is data merging of micro-scale in-situ data (AQ, spectrometry) with satellite data (especially new upcoming launches like TEMPO, but this could also work for even the older satellites like Terra, who's instruments provide only synoptic/regional scale resolution).  I think STELLA could be one way to do this, and it makes me super excited. 😊  Anyone have additional thoughts or ideas for air quality and STELLAAA??  

Also, on a somewhat related note, Lance Hubbard, a scientist from PNNL has created a new biotechnology that detects COVID in the air using micelles –[ more info here.  ](https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.pnnl.gov%2Fnews-media%2Fcovid-causing-virus-air-detected-high-tech-bubbles&data=05%7C01%7Cnyssa.rayne%40nasa.gov%7C60d372b799b14e095acc08db26ffc83d%7C7005d45845be48ae8140d43da96dd17b%7C0%7C0%7C638146652232807807%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=u888A71fJLpcHFUMu59GC%2FarzDm9WQLt5qXYrrYY0rY%3D&reserved=0)It's still fairly expensive (~ commercial systems:  $20k-$50k per system with ~5k-20k yearly subscription fees; Hubbard's new tech: ~$3-15k/system, depending mostly on the air handling interface and electronics).   

 If we could make this type of product in a cheaper way, we could literally solve so many AQ problems all at once.

**Comment by paulmirel** [comment]
I think the STELLA-Q platform could be adapted for measuring air quality. Omit the spectral sensors, add CO2 and PM2.5 and maybe VOC. STELLA-Q can be assembled by middle schoolers, it's just connectors. 

I like the SCD-40 for CO2, the PMSA003I for PM2.5 (also gives PM1 and PM10!) and SGP-40 for VOC. $110 for the sensors on top of the STELLA-Q core components, less the STELLA-Q sensors. Not a cheap instrument, but one with believable data, which is entirely the point of measuring. 

Thanks for your enthusiasm for STELLA! 

P

**Comment by Landsat-Mike** [comment]
Is there a cheap way of calibrating these AQ sensors? If so how would we go about it? From what I understand AQ sensors are to be calibrated every 6 months.
